packages/@expo/cli/src/start/server/metro/MetroBundlerDevServer.ts

```diff
@@ -813,7 +813,8 @@ export class MetroBundlerDevServer extends BundlerDevServer {
           // TODO: Add a less leaky version of this across the framework with just [key, value] (module ID, chunk).
           Object.fromEntries(
             Array.from(ssrManifest.entries()).map(([key, value]) => [
-              path.join(serverRoot, key),
+              // Must match babel plugin.
+              './' + path.relative(this.projectRoot, path.join(serverRoot, key)),
```

Comment:
This creates forward slashes on Windows, probably best to wrap the `path.relative` function in the `toPosixPath` helpers.

![image](https://github.com/user-attachments/assets/23dd2d97-e35c-4435-8d2b-dcbfdbb656ef)